src/app/game/game-intermediate-result/game-example-drawings/game-example-drawings.component.ts

```diff
@@ -0,0 +1,63 @@
+import { Component, OnInit } from '@angular/core';
+import { ExampleDrawingService } from '../../services/example-drawing.service';
+import { SpeechBubbleComponent } from '../../speech-bubble/speech-bubble.component';
+import { ArrowAlignment, PointerSide, SupportedLanguages } from '@/app/shared/models/interfaces';
+import { CustomColorsIO } from '@/app/shared/customColors';
+import { TranslatePipe } from '@/app/core/translation.pipe';
+import { DrawingService } from '../../services/drawing.service';
+import { TranslationService } from '@/app/core/translation.service';
+import * as NO from '../../../../assets/translation/NO.json';
+import * as EN from '../../../../assets/translation/EN.json';
+
+@Component({
+  selector: 'app-game-example-drawings',
+  standalone: true,
+  imports: [SpeechBubbleComponent, TranslatePipe],
+  templateUrl: './game-example-drawings.component.html',
+  styleUrl: './game-example-drawings.component.scss',
+})
+export class GameExampleDrawingsComponent implements OnInit {
+  language: SupportedLanguages = this.translationService.getCurrentLang();
+  exampleDrawings: string[] = [];
+  label = '';
+  PointerSide = PointerSide;
+  ArrowAlignment = ArrowAlignment;
+  CustomColorsIO = CustomColorsIO;
+  constructor(
+    private translationService: TranslationService,
+    private drawingService: DrawingService,
+    private exampleDrawingService: ExampleDrawingService
+  ) {}
+  ngOnInit(): void {
+    this.label = this.drawingService.label;
+    this.exampleDrawingService.getExampleDrawings(3, this.label, this.language).subscribe((res) => {
+      this.exampleDrawings = res;
+    });
+  }
+
+  getMessage(): string {
+    const hasCorrectGuess = this.drawingService.lastResult.hasWon;
+    return this.getTranslatedMessage(this.label, this.language, hasCorrectGuess);
+  }
+
+  getTranslatedMessage(label: string, lang: SupportedLanguages, hasCorrectGuess: boolean): string {

```

Comment:
Shouldn't translation be handled by translate pipe instead? For example like in game-drawing-display html component:
`<p class="round-score-text">+ {{ roundScore }} {{ 'POINTS' | translate }}</p>`